kura/org.eclipse.kura.core.configuration.test/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
I like this test fragment.
@markoer I had exactly this in mind for #785. Can you please review?